tests/acceptance/core/testcase/ingressdns.go

```diff
@@ -0,0 +1,48 @@
+package testcase
+
+import (
+	"github.com/k3s-io/k3s/tests/acceptance/core/service/assert"
+	"github.com/k3s-io/k3s/tests/acceptance/shared/util"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestIngress(deployWorkload bool) {
+	if deployWorkload {
+		_, err := util.ManageWorkload("create", "ingress.yaml", *util.Arch)
+		Expect(err).NotTo(HaveOccurred(),
+			"Ingress manifest not deployed")
+	}
+
+	err := assert.ValidateOnHost(GetIngressRunning+util.KubeConfigFile, Running)
```

Comment:
This is one of many suggestions I could make. `GetIngressRunning` is used exlusively in this funciton. There is no point in making it package wide accessible variable. 
```suggestion
    cmd := "kubectl get pods  -l k8s-app=nginx-app-ingress --field-selector=status.phase=Running  --kubeconfig="
	err := assert.ValidateOnHost(cmd+util.KubeConfigFile, Running)
```